lib/features/weather/model/weather_view_model_state.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter_training/features/weather/model/weather_condition.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'weather_view_model_state.freezed.dart';
+
+@freezed
+class WeatherViewModelState with _$WeatherViewModelState {
+  const factory WeatherViewModelState({
+    required WeatherCondition? weatherCondition,
+    required int? maxTemperature,
+    required int? minTemperature,
```

Comment:
[imo]
`required` ない方が初期値の値で `null` を明示しなくてよくなるため、個人的にはおすすめです 

```suggestion
    WeatherCondition? weatherCondition,
    int? maxTemperature,
    int? minTemperature,
```

↓ のようにかけます

```dart
//...
@riverpod
class WeatherViewModel extends _$WeatherViewModel {
  @override
  WeatherViewModelState build() => const WeatherViewModelState();
//...
```

Reply:
ありがとうございます! 修正しました
9f28e5d00368a16eb1aa2858ecce437b4a1c58d0